iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/ops/IcebergCatalogWrapper.java

```diff
@@ -102,10 +112,15 @@ public IcebergCatalogWrapper(IcebergConfig icebergConfig) {
             key -> catalogPropertiesToClientKeys.contains(key));
 
     this.catalogPropertiesMap = icebergConfig.getIcebergCatalogProperties();
-  }
 
-  public IcebergCatalogWrapper() {
-    this(new IcebergConfig(Collections.emptyMap()));
```

Comment:
We can still maintain this default constructor like:

```java

public IcebergCatalogWrapper() {
  this(xxxx, false)
}
```

What do you think?

Reply:
it's used to test only, seems useless to maintain the default constructor